src/platform/Ameba/KeyValueStoreManagerImpl.cpp

```diff
@@ -49,19 +49,22 @@ CHIP_ERROR KeyValueStoreManagerImpl::_Get(const char * key, void * value, size_t
         return (err = CHIP_ERROR_NOT_IMPLEMENTED);
     }
 
-    ret = getPref_bin_new(key, key, (uint8_t *) value, value_size, read_bytes_size);
-    switch (ret)
+    if (read_bytes_size)
     {
-    case 0:
-        return CHIP_NO_ERROR;
-    case -6:
-        return CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND;
-    case -7:
-        return CHIP_ERROR_INVALID_ARGUMENT;
-    case -8:
-        return CHIP_ERROR_BUFFER_TOO_SMALL;
-    default:
-        break;
+        ret = getPref_bin_new(key, key, (uint8_t *) value, value_size, read_bytes_size);
```

Comment:
I think it should still be possible to read the value even if `read_bytes_size` is nullptr. Just the actual size should not be reported.

Reply:
This has been changed, actual size will not be reported if read_bytes_size is null.